src/components/Checkout/OrderTotals.vue

```diff
@@ -63,7 +63,7 @@
 			</kv-tooltip>
 		</div>
 
-		<div class="order-total">
+		<div class="order-total" data-test="total-due">
 			<strong>
 				<template v-if="!showPromoCreditTotal">Total: </template>
```

Comment:
@mcstover, David is wondering if we can keep these both as "Total Due" for consistency